book/src/SUMMARY.md

```diff
@@ -21,6 +21,7 @@
   - [Persistence of group state](user_manual/persistence.md)
   - [Credential validation](user_manual/credential_validation.md)
   - [WebAssembly](user_manual/wasm.md)
+  - [Examining a welcome message](user_manual/examining_welcome_message.md)
```

Comment:
There is the `Join a group from a Welcome message` chapter. I think this could fit in there. Because it's the negative case of processing the welcome.